pkg/sysregistriesv2/system_registries_v2_test.go

```diff
@@ -246,11 +259,16 @@ func assertRegistryLocationsEqual(t *testing.T, expected []string, regs []Regist
 	for _, r := range regs {
 		names = append(names, r.Location)
 	}
+	sort.Strings(expected)
```

Comment:
Update (or drop) the “and their order” comment above, please.

Reply:
Order is now enforced again, so I dropped the sorting.